sql/core/src/test/resources/sql-tests/inputs/table-valued-functions.sql

```diff
@@ -26,5 +26,5 @@ select * from RaNgE(2);
 EXPLAIN select * from RaNgE(2);
 
 -- cross-join table valued functions
-set spark.sql.crossJoin.enabled=true;
-EXPLAIN EXTENDED select * from range(3) cross join range(3);
+SET spark.sql.crossJoin.enabled=true;
```

Comment:
Could you remove this line? If we specify `CROSS JOIN` in the query, no need to set this parm.